lib/srv/sess.go

```diff
@@ -239,7 +239,8 @@ func (sc *sudoersCloser) Close() error {
 // file, if any. If the returned closer is not nil, it must be called at the
 // end of the session to cleanup the sudoers file.
 func (s *SessionRegistry) TryWriteSudoersFile(identityContext IdentityContext) (io.Closer, error) {
-	if s.sudoers == nil {
+	sudoWriter := s.Srv.GetHostSudoers()
```

Comment:
This is to make sure `TryWriteSudoersFile()` always does the correct thing based on the server configuration it's deployed to. There may not be a reason to cache `sudoers` on `sessionRegistry` at all to be honest, so I might change that in a future PR as well